src/udev/udevd.c

```diff
@@ -1566,7 +1566,7 @@ static int help(void) {
                "  -d --daemon                 Detach and run in the background\n"
                "  -D --debug                  Enable debug output\n"
                "  -c --children-max=INT       Set maximum number of workers\n"
-               "  -e --exec-delay=SECONDS     Seconds to wait before executing RUN=\n"
+               "  -e --exec-delay=SECONDS     Seconds to wait before executing each of RUN{program} instructions\n"
```

Comment:
This is longer than 80 columns and will fail. I think it's better to keep the
original. `--help` is supposed to be a quick reminder, not reference docs.